tests/ValueObjectTest.php

```diff
@@ -112,5 +112,17 @@ public function test()
         ];
         $actual = $fakeRecord->getArrayCopy();
         $this->assertEquals($expect, $actual);
+
+
+        // Test new entity
+        $newFakeEntity = new Fake(new Email('fake@example.com'),
```

Comment:
Your changes are related to the `EntityHandler` but this is the test class for a value object. Is this correct?

Reply:
Yes because it's on the persist of a ValueObject.

Reply:
Two suggestions for the test:

1. Create a separate test method and used a name that describes the current testing process.
2. At the moment the test did not perform any assertions, that should be changed.

Reply:
Done ;)